main/solution/ui/src/models/hello/HelloStore.js

```diff
@@ -25,7 +25,7 @@ import { Hello } from './Hello';
 // ==================================================================
 const HelloStore = BaseStore.named('HelloStore')
   .props({
-    hellos: types.array(Hello),
+    hellos: types.optional(types.array(Hello), []),
```

Comment:
nit: how is this file related to the change?

Reply:
Not related to the fix, but a week ago the lobx-state-tree version was updated which caused local debugging to fail. This is the fix for that.